README.md

```diff
@@ -7,26 +7,27 @@ Basic usage and installation is described below. [Click here for complete Sophos
 ## Usage
 
 ```shell
-$ factoryctl --help   
 Usage: factoryctl <command> [options]
 
 Commands:
   factoryctl run             Execute pipeline or job
   factoryctl rerun [run-id]  Reschedule pipeline run
-  factoryctl delete          Delete specified resource
   factoryctl list            List specified resources
   factoryctl get             Get specified resource
   factoryctl create          Create specified resource
+  factoryctl delete          Delete specified resource
 
 Options:
-      --version     Show version number                                                                        [boolean]
-  -v, --verbose     Print detailed output                                                                      [boolean]
-      --format      Output format                                    [choices: "wide", "json", "yaml"] [default: "wide"]
-      --filter      Filter output using JsonPath                                                                [string]
-      --address     Address of the Sophos Factory API server
-                    [string] [default: FACTORY_ADDRESS environment variable if set, otherwise https://api.dev.factory.sophos.com/v1]
-      --auth-token  Authentication token                     [string] [default: FACTORY_AUTH_TOKEN environment variable]
-      --help        Show help                                                                                  [boolean]
+      --version       Show version number                                                                      [boolean]
+      --address       Address of the Sophos Factory API server
+                             [string] [default: FACTORY_ADDRESS if set, otherwise https://api.dev.factory.sophos.com/v1]
+      --auth-address  Address of the Sophos Factory Auth API server
```

Comment:
Added as a separate parameter, though that was not strictly necessary since only one API will be used in a single command.